### PR TITLE
Make install script auto-detect arch

### PR DESCRIPTION
**Pull request by mcm001**

### CODE REVIEW

File: scripts/install.sh [hunk 1]
```diff
@@ -5,32 +5,44 @@ if [ "$(id -u)" != "0" ]; then
...
-apt-get install --yes cpufrequtils
-echo "cpufrequtils installation complete."
-
-echo "Setting cpufrequtils to performance mode"
-if [ -f /etc/default/cpufrequtils ]; then
-    sed -i -e 's/^#\?GOVERNOR=.*$/GOVERNOR=performance/' /etc/default/cpufrequtils
-else
-    echo 'GOVERNOR=performance' > /etc/default/cpufrequtils
-fi
-
```
Comment (mdurrani808) [inline scripts/install.sh:28]: Why is this all being removed?

File: scripts/install.sh [hunk 2]
```diff
@@ -5,32 +5,44 @@ if [ "$(id -u)" != "0" ]; then
...
 echo "Installing the JDK..."
 if [ $(dpkg-query -W -f='${Status}' openjdk-11-jdk-headless 2>/dev/null | grep -c "ok installed") -eq 0 ];
 then
-   apt-get update
-   apt-get install --yes openjdk-11-jdk-headless
+   apt update
+   apt-get install openjdk-11-jdk-headless;
```
Comment (mdurrani808) [inline scripts/install.sh:45]: Shouldn't this be jdk 17 now?

File: scripts/install.sh (hunk 1 above)
Comment (mcm001) [reply scripts/install.sh:28]: No clue, git moment. Fixing now

File: scripts/install.sh (hunk 2 above)
Comment (mcm001) [reply scripts/install.sh:45]: Not required yet, we still target Java 11 language support. And I don't want to change it coz we don't have time to fully test Java 17 right now.

**Review by gerth2 — APPROVED** [review]

**Pull request merged**